packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json

```diff
@@ -25,7 +25,7 @@
   "pci_additional_ip_floating_ip": "IP flottantes (Public Cloud)",
   "pci_additional_ip_floating_ip_description": "Solution optimisée pour les cas d’utilisation Public Cloud, l’automatisation (Terraform) et le support de réseaux privés régionalisés. Disponible en modèle de facturation à l’usage.",
   "pci_additional_ip_create_step_select_region_floating_ip": "Sélectionnez une région pour votre nouvelle IP flottante",
-  "pci_additional_ip_create_step_select_region_description_floating_ip": "Vous ne trouvez pas la région que vous désirez dans la liste ? Pour en savoir plus sur la disponibilité des produits par région, <a href=\"{{ guideLink }}\" target=\"_blank\">cliquez ici.</a>",
+  "pci_additional_ip_create_step_select_region_description_floating_ip": "Vous ne trouvez pas la région que vous désirez dans la liste ? Pour en savoir plus sur la disponibilité des produits par région, <a href=\"{{ guideLink }}\" target=\"_blank\" data-track-name=\"{{ trackLabel }}\" data-track-on=\"click\" data-track-type=\"action\">cliquez ici.</a>",
```

Comment:
Is it possible to remove the HTML from the translation ?

Reply:
Since the click needs to be tracked, we cannot remove HTML from translation files.